tools/maxquant/mqwrapper.py

```diff
@@ -74,7 +74,8 @@
     mqpar_in = mqpar_temp
 mqpar_out = args['mqpar_out'] if args['mqpar_out'] != 'None' else mqpar_temp
 
-m = mqparam.MQParam(mqpar_out, mqpar_in, args['exp_design'],
+exp_design = args['exp_design'] if args['exp_design'] != 'None' else None
+m = mqparam.MQParam(mqpar_out, mqpar_in, exp_design,

```

Comment:
That's why I made this change. Should work, shouldn't it?

Reply:
If it's really 'None' that is passed through yes, it's a little bit unusual to pass None through the command line but yes, this should work.